locations/spiders/denmans_gb.py

```diff
@@ -0,0 +1,49 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class DenmansGBSpider(Spider):
+    name = "denmans_gb"
+    item_attributes = {"brand": "Denmans", "brand_wikidata": "Q116508855"}
+    # This seems to return all stores regardless of lat-long; as long as it's in the UK?
+    start_urls = ["https://www.denmans.co.uk/den/store-finder/findNearbyStores?latitude=51&longitude=-0"]
+
+    def parse(self, response):
+        for store in response.json()["results"]:
+            store["address"]["street_address"] = ", ".join(
+                filter(
+                    None,
+                    [
+                        store["address"].pop("line1"),
+                        store["address"].pop("line2"),
+                        store["address"].pop("line3"),
+                    ],
+                )
+            )
+            store["ref"] = store.pop("name")
+            item = DictParser.parse(store)
+            item["phone"] = store["address"]["phone"]
```

Comment:
Should we move this before DictParser then?

Reply:
I don't think it's massively important either way.